### PR TITLE
Update template for v1.0

### PR DESCRIPTION
**Pull request by JTeijema**
This PR will update this template to v1.0 of ASReview. Includes the fix for issue #5.

### CODE REVIEW

**Comment by JTeijema** [comment]
After installing this template, the list classifiers class shows the newly installed classifier. 
```powershell
>>> from asreview.models import classifiers
>>> for i in classifiers.list_classifiers(): print(i)
<class 'asreview.models.classifiers.logistic.LogisticClassifier'>
<class 'asreview.models.classifiers.lstm_base.LSTMBaseClassifier'>
<class 'asreview.models.classifiers.lstm_pool.LSTMPoolClassifier'>
<class 'asreview.models.classifiers.nb.NaiveBayesClassifier'>
<class 'asreview.models.classifiers.nn_2_layer.NN2LayerClassifier'>
<class 'asreview.models.classifiers.rf.RandomForestClassifier'>
<class 'asreview.models.classifiers.svm.SVMClassifier'>
<class 'asreviewcontrib.models.nb_default_param.NaiveBayesDefaultParamsModel'>
```

**Comment by JTeijema** [comment]
Moreover, the classifier is shown in the front end too. 
image

Considering this, I can confirm the template to work with v1.0

File: README.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 # Template for extending ASReview with new model
 
-ASReview has support for extensions, which enable you to seemlessly integrate
+<a href="https://github.com/asreview/asreview"><img src="https://img.shields.io/badge/ASReview-v1.0-%23ffcb05" alt="ASReview v1.0" /></a>
```
Comment (J535D165) [inline README.md:3]: Can you use markdown here? What is the added value btw of this badge? It works with 0.x versions as well.

**Review by J535D165 — COMMENTED** [review]
Thanks for all the testing!

File: README.md (hunk 1 above)
Comment (JTeijema) [reply README.md:3]: My main motivation here was that I like badges.

Comment (JTeijema) [reply README.md:3]: I meant it to show that it's 1.0 compatible.

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Missed this one, merged and thanks :)

**Pull request merged**